parameter:
  ros__parameters:
  
    interlace: 
    hirizonal_interlace: 
    vertical_interlace: 
    ip:
    port:
    frame_id:
    imu_frame_id:
    mag_frame_id:
    range_min:
    auto_reset:
    allow_jump_back:
    output_cycle:
